packages/syft/src/syft/__init__.py

```diff
@@ -59,6 +59,7 @@
 from syft.core.node.vm.vm import VirtualMachineClient  # noqa: F401
 from syft.core.plan.plan import Plan  # noqa: F401
 from syft.core.plan.plan_builder import make_plan  # noqa: F401
+from syft.core.tensor.tensor import Tensor  # noqa: F401
```

Comment:
This showcased a lot of problems for the allowed methods.

Reply:
What errors george, is it resolved?

Reply:
Now it works -- 7 days ago there were some ```allowlist``` related problems